identity/src/keypair.rs

```diff
@@ -39,13 +39,27 @@ use crate::proto;
     feature = "ed25519",
     feature = "rsa"
 ))]
+use hkdf::Hkdf;
+#[cfg(any(
+    feature = "ecdsa",
+    feature = "secp256k1",
+    feature = "ed25519",
+    feature = "rsa"
+))]
```

Comment:
Can we ditch these in favor of fully qualifying them at the usage site? Would save as quite a few LoC.